dateparser/parser.py

```diff
@@ -24,6 +25,10 @@ def no_space_parser_eligibile(datestring):
 
     return False
 
+def is_8_digit_number(datestring):
+    if EIGHT_DIGIT.match(datestring) is None:
+        return False
+    return True
```

Comment:
This could be simplified as `return bool(EIGHT_DIGIT.match(datestring))`, so it may not be worth having a separate function for this.

Reply:
Thanks for the suggestion. Updated it.